### PR TITLE
Fixes #15127: Positioning fixed for menubar, top_padding for body

### LINKED ISSUE #15127: [UI] Horrible white square
### Describe the bug

image

This occurs for some reason a some time but is very annoying ...
Use designer and navigate between pages

### PR DESCRIPTION
**Pull request by ankush-0x90**
### Description
The white guest block which was appearing after reloading on designer's page

Please describe your pull request.
Added padding to body and positioning to #floating-navbar

Fixes #15127 

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

**Review by williamdes — APPROVED** [review]
LGTM, I need to test it

**Comment by asprazz** [comment]
Sure. I know the seriousness :100: after playing game with opening and closing issue :))

**Comment by asprazz** [comment]
@sudo-bot Deploy PR

**Comment by williamdes** [comment]
@asprazz I will try to fix the bot, and deploy :)

@sudo-bot deploy pr

**Comment by williamdes** [comment]
@asprazz :tada: fixed !
very good idea to ask a deploy, I added you to the trusted list of deploy users :)

**Comment by asprazz** [comment]
> @asprazz  fixed [sudo-bot/gh-deployer@ec2c451](https://github.com/sudo-bot/gh-deployer/commit/ec2c451926a246a863d30b7ebe1902e8f707c1e2) !
> very good idea to ask a deploy, I added you to the trusted list of deploy users :)

The idea is so great that I tried it here :) Thank you for adding me

**Comment by williamdes** [comment]
> The idea is so great that I tried it here :) Thank you for adding me

Thank you so much !

Your PR seems to work fine, will merge it

File: js/navigation.js [hunk 1]
```diff
@@ -1128,8 +1128,26 @@ var ResizeHandler = function () {
...
+        $('#floating_menubar')
+            .css('margin-' + this.left, $('#pma_navigation').width() + $('#pma_navigation_resizer').width())
+            .css(this.left, 0)
+            .css({
+                'position': 'fixed',
+                'top': 0,
+                'width': '100%',
+                'z-index': 99
+            })
+            .append($('#serverinfo'))
```
Comment (williamdes) [inline js/navigation.js]: @asprazz why did you need to do this ?

File: js/navigation.js [hunk 2]
```diff
@@ -1128,8 +1128,26 @@ var ResizeHandler = function () {
         var windowWidth = $(window).width();
         $('#pma_navigation').width(pos);
         $('body').css('margin-' + this.left, pos + 'px');
-        // Issue #15127
-        $('#floating_menubar, #pma_console')
+        // Issue #15127 : Adding fixed positioning to menubar
+        $('#floating_menubar')
+            .css('margin-' + this.left, $('#pma_navigation').width() + $('#pma_navigation_resizer').width())
+            .css(this.left, 0)
+            .css({
```
Comment (williamdes) [inline js/navigation.js]: I think you are obliged to do so ?

File: js/navigation.js (hunk 1 above)
Comment (asprazz) [reply js/navigation.js]: I think we don't need to append anything as it is already present in the DOM
I referenced a code in repo for this issue
Will fix this

File: js/navigation.js (hunk 2 above)
Comment (asprazz) [reply js/navigation.js]: I thought this after comparing between those two pages (the page without space & the page with space)

Comment (asprazz) [reply js/navigation.js]: I didn't find if any better solution to this other than adding padding to menubar though.. :(

File: js/navigation.js (hunk 1 above)
Comment (williamdes) [reply js/navigation.js]: Thank you

Comment (williamdes) [reply js/navigation.js]: Seems like you removed them from the wrong place in 925db3ee22b7fbf8859fb164d88db0905360aa32

**Comment by williamdes** [comment]
@asprazz on next push can you squash commits ?

**Review by williamdes — APPROVED** [review]
:100:

**Comment by asprazz** [comment]
> @asprazz on next push can you squash commits ?

I tried squashing Did it work as you expected?

**Comment by williamdes** [comment]
> I tried squashing Did it work as you expected?

Yes, perfect

One last review and I push the merge 

Thanks for your contribution !

**Comment by asprazz** [comment]
> > I tried squashing Did it work as you expected?
> 
> Yes, perfect
> 
> One last review and I push the merge
> 
> Thanks for your contribution !

Can you wait just 1 sec I will have to add a semicolon

**Comment by williamdes** [comment]
Hi @asprazz have a look to #15581 
Now we know that appending was needed

**Comment by asprazz** [comment]
@williamdes I just saw the fix, I didn't analyse at first the root of the problem @ShailuJain did great work.. :+1:

**Pull request merged**